### ISSUE TITLE
Save Loop on/off, Loop position and pinned values.

### ISSUE THREAD

**Issue by ScreenDream** [tixl3d/tixl#188]
We recently had to restart t3 several times and each time the keyframe editor was back to it's default state, which became tedious fast.

So it would be great if:
a.) the Loop on/off state would be saved.
b.) the Loop start and end points would be saved.
c.) pinned animation lanes would be saved.

Basically what we're after is that after returning to a project, it should be in the same state as when we last left/saved it.

Thank you very much!

**Comment by pixtur** [comment]
Thanks for the suggestion. There are multiple ways of persisting these settings:

1. For all GraphWindows and Times lines in UserSettings
2. For each individual "project" op similar to Soundtrack
3. For every Operator instance.

I guess option 2 would make most sense. What are your thoughts on that?

**Comment by ScreenDream** [comment]
Yes, I think 2. is the way to go, since each project has a different loop length, different pinned lanes etc.

Thank you!

Cheers,

Tom

**Comment by ScreenDream** [comment]
Looking back into Tool 3.7, this is still one of the things I miss each time I open a project.
My hope would be, to one day come back to a project and have it all configured like when I left it.
This would also include things like "Render Sequence" and "Render Video" settings (if you iterate over a certain animation, ATM you have to set this manually every time).
Getting there... ;-)

**Comment by pixtur** [comment]
Thanks for bringing this back to attention

**Comment by pixtur** [comment]
Duplicates #715
